Sources/SwiftInspectorVisitors/Tests/FileVisitorSpec.swift

```diff
@@ -99,6 +100,37 @@ final class FileVisitorSpec: QuickSpec {
           expect(matchingStructs.count) == 1
         }
 
+        it("finds TestStruct.InnerClass") {
+          let matchingStructs = self.sut.fileInfo.classes.filter {
+            $0.name == "InnerClass"
+              && $0.parentTypeName == "TestStruct"
+          }
+          expect(matchingStructs.count) == 1
```

Comment:
doubling down on these tests given the conversation in https://github.com/fdiaz/SwiftInspector/pull/65#discussion_r567043585. We can always revisit if I'm wrong 😅